presto-cassandra/src/test/java/com/facebook/presto/cassandra/CassandraQueryRunner.java

```diff
@@ -54,7 +56,12 @@ public static synchronized DistributedQueryRunner createCassandraQueryRunner()
                     com.datastax.driver.core.Session session = cluster.connect()) {
                 createOrReplaceKeyspace(session, "tpch");
             }
-            copyTpchTables(queryRunner, "tpch", TINY_SCHEMA_NAME, createCassandraSession("tpch"), TpchTable.getTables());
+            List<TpchTable<?>> tables = TpchTable.getTables();
+            copyTpchTables(queryRunner, "tpch", TINY_SCHEMA_NAME, createCassandraSession("tpch"), tables);
+            for (TpchTable table : tables) {
+                EmbeddedCassandra.flush("tpch", table.getTableName());
+            }
+            EmbeddedCassandra.refreshSizeEstimates();
```

Comment:
How long does this take? Is someone likely to run across it when using the connector?